fedimint-cli/Cargo.toml

```diff
@@ -25,7 +25,7 @@ anyhow = { workspace = true }
 async-trait = { workspace = true }
 base64 = "0.22.0"
 bip39 = { version = "2.0.0", features = ["rand"] }
-bitcoin = "0.30.2"
+bitcoin = { workspace = true }
```

Comment:
Note for reviewer: Unrelated but I noticed I missed this in a previous migration PR